vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -179,6 +179,56 @@ function Test-FilesToCacheOnVHD
 
 }
 
+function Test-ValidateAllSignature {
+    foreach ($dir in $map.Keys) {
+        # Skip the validation of k8s pacakges since win-bridge.exe is not signed yet
+        if ($dir.StartsWith("c:\akse-cache\win-k8s\")) {
```

Comment:
Could we only ignore win-birdge.exe in k8s packages?

Reply:
Add unsigning files in an array and throw an error in the end if it is not empty after removing the files with known issues.

Reply:
Thanks for the comment. Resolved it.